### PR TITLE
fix Notification getContainer config bug

### PR DESCRIPTION
**Pull request by hardfist**
Change-Id: I8eb84c640bb75c7e3b7c9c35b3105ca147888d54

First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you propose PR to correct branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by hardfist** [comment]
fix issue #6036

File: components/notification/__tests__/placement.test.js [hunk 1]
```diff
@@ -140,4 +140,17 @@ describe('Notification.placement', () => {
...
+    const $container = document.createElement('div');
+    document.body.appendChild($container);
+    config({
+      top: 50,
+      bottom: 100,
+      getContainer() {
+        return $container;
+      },
+    });
+    expect($container.querySelector('.ant-notification')).not.toBe(null);
```
Comment (benjycui) [inline components/notification/__tests__/placement.test.js]: I don't understand, I think you should open a new notification before `expect`

Comment (benjycui) [inline components/notification/__tests__/placement.test.js]: But CI passed... Why?

Comment (benjycui) [inline components/notification/__tests__/placement.test.js]: Oops, I make a mistake..

**Pull request merged**